### PR TITLE
Fixup tests

### PR DESCRIPTION
**Pull request by PokeCodec**
### Description of the changes:
- Removed unneeded "ToString" calls
- Fixed typos
- Renamed "fEButtonState" to "FEButtonState"

### How changes were validated:

- All tests passed.

### CODE REVIEW

File: src/CalculatorUITestFramework/NumberPad.cs [hunk 1]
```diff
@@ -29,7 +30,7 @@ public class NumberPad
         /// <param name="number">Number to be entered into the calculator.</param>
         public void Input(double number)
         {
-            string numberStr = number.ToString();
+            string numberStr = number.ToString(CultureInfo.CurrentCulture);
```
Comment (rudyhuyn) [inline src/CalculatorUITestFramework/NumberPad.cs:33]: Thank you for this PR, the culture should be "CultureInfo.InvariantCulture", to make sure that we can correctly parse it after

**Review by rudyhuyn — CHANGES_REQUESTED** [review]

File: src/CalculatorUITestFramework/NumberPad.cs (hunk 1 above)
Comment (PokeCodec) [reply src/CalculatorUITestFramework/NumberPad.cs]: Done!

**Comment by PokeCodec** [comment]
Ready for review @rudyhuyn

**Review by mcooley — APPROVED** [review]

**Comment by PokeCodec** [comment]
Ready for review @rudyhuyn

**Review by joseartrivera — APPROVED** [review]
Thanks for the PR, looks good!

**Pull request merged**